File: Array/twoSum/twosum.c
#include <ctype.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#define BASE 1024
int* twoSum(int* nums, int target);


int main(){
    int num_list[] = {3,22,2,2,3};
    int target = 6;

    int* finalArray = twoSum(num_list, target);
    printf("First: %i\nSecond: %i\n", finalArray[0], finalArray[1]);
    free(finalArray);
    return 0;
}

int* twoSum(int* nums, int target){
    int numsSize = sizeof(nums);
    int returnSize = 2;
    int *returnArray = (int*)malloc(returnSize * sizeof(int*));
    int temp = 0;
    for (int i = 0; i<numsSize; i++){
        temp = i;
        for (int j = temp+1; j<numsSize;j++){
            if (nums[temp] + nums[j] == target){
                returnArray[0] = temp;
                returnArray[1] = j;
                return(returnArray);
            }
        }
    }   
    return(NULL);
}

File: Array/twoSum/twoSum.py
class Solution:
    def twoSum(self, nums, target):
        dict0 = {}
        for index,num in enumerate(nums):
            if target-num in dict0:
                return([index, dict0[target-num]])
            else:
                dict0[num] = index

def main():
    _solution = Solution()
    finalList = _solution.twoSum([2,7,11,15], 9)
    print(finalList)
    
if __name__ == "__main__":
    main()